configuration.rst

```diff
@@ -936,9 +936,34 @@ Storing Environment Variables In Other Files
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 By default, the environment variables are stored in the ``.env`` file located
-at the root of your project. However, you can store them in other file by
-setting the ``SYMFONY_DOTENV_PATH`` environment variable to the absolute path of
-that custom file.
+at the root of your project. However, you can store them in other file in
+multiple ways:
+
+* By calling the :method:`Symfony\\Component\\Dotenv\\Dotenv::bootEnv`
```

Comment:
I am actually no 100% sure. Is it in `src/Kernel.php`? :thinking:

Reply:
This is only when the dotenv component is used standalone, or the symfony runtime is not used: https://github.com/symfony/demo/blob/v1.6.0/public/index.php#L10

Other solutions are enabled by symfony runtime: https://github.com/symfony/symfony/blob/4266345e5c53c128b0d0b0dcfbce1143c52c52e5/src/Symfony/Component/Runtime/SymfonyRuntime.php#L104-L107

Reply:
What do you think of the new wording?

Reply:
Instead of having a note below, the condition for using this method should be here. 

You can create 2 paragraphs: "if you use the Runtime component, the path is part of the options you can set in composer.json or in the environment variable. Otherwise, if you invoke the dotenv class directly in your bootstrap.php or any other file of your application, you can customize the .env path."
Voilà, in summary.

Reply:
Sorry it took that long. I removed the unordered list and replaced with a few paragraphs.